### PR TITLE
feat: support wearable.json

### PR DESCRIPTION
**Pull request by nicosantangelo**
Ditch `asset.json` jsons in favour of the new (alpha) `wearable.json` file.

Examples:
[portable.zip](https://github.com/decentraland/builder/files/9304295/portable.zip)
[banana.zip](https://github.com/decentraland/builder/files/9304298/banana.zip)
[hat.zip](https://github.com/decentraland/builder/files/9304301/hat.zip)

### CODE REVIEW

File: src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx [hunk 1]
```diff
@@ -44,58 +43,34 @@ export default class ImportStep extends React.PureComponent<Props, State> {
...
-
-    // asset.json contains data to populate parts of the state
-    const assetJsonPath = fileNames.find(path => basename(path) === ASSET_MANIFEST)
     let assetJson: ItemAssetJson | undefined
-
-    if (assetJsonPath) {
-      const assetRaw = zip.file(assetJsonPath)
-      const content = await assetRaw.async('text')
-      assetJson = JSON.parse(content)
+    let modelPath: string | undefined
```
Comment (LautaroPetaccio) [inline src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx:51]: I think we shouldn't keep the concept of `assetJson` and its type, as it refers to the older mechanism of importing assets.

File: src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx [hunk 2]
```diff
@@ -44,58 +43,34 @@ export default class ImportStep extends React.PureComponent<Props, State> {
...
-      contents[file.name] = file.blob
-      return contents
-    }, {})
-
     if (!modelPath) {
       throw new MissingModelFileError()
     }
 
-    const result = await this.processModel(modelPath, contents)
+    const result = await this.processModel(modelPath, content)
```
Comment (LautaroPetaccio) [inline src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx:71]: The result of `processModel` will ignore one of the representations, I think we should change the code so we can return from this function all the data we can to use it later when building the item.

**Review by LautaroPetaccio — APPROVED** [review]

File: src/modules/item/utils.ts [hunk 3]
```diff
@@ -296,7 +300,7 @@ export function getThumbnailURL(item: Item) {
 }
 
 export function getRarities() {
-  return Object.values(ItemRarity)
+  return ItemRarity.schema.enum as ItemRarity[]
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply src/modules/item/utils.ts:303]: I think enum is already the `Object.values` version of the enum: https://github.com/decentraland/common-schemas/blob/main/src/dapps/rarity.ts#L20

File: src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx (hunk 1 above)
Comment (nicosantangelo) [reply src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx:51]: Maybe we can expand ModelData (the return type) to return all the properties we currently have on assetJSON which are:

```ts
name
description
rarity
category
bodyShape
```

another option is to just rename the prop?

I'm hesitant to rewrite everything as this also is being checked later on in `getAssetJsonProps`

What did you have in mind?

File: src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx (hunk 2 above)
Comment (nicosantangelo) [reply src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx:71]: Sorry a bit confused here. I think process model does not deal with representations?

File: src/modules/item/utils.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (LautaroPetaccio) [reply src/modules/item/utils.ts:304]: You're right, my mistake!

**Pull request merged**